pkg/util/helper/workstatus.go

```diff
@@ -75,7 +89,11 @@ func AggregateClusterResourceBindingWorkStatus(c client.Client, binding *workv1a
 		return err
 	}
 
-	if reflect.DeepEqual(binding.Status.AggregatedStatus, aggregatedStatuses) {
+	currentBindingStatus := workv1alpha2.ResourceBindingStatus{AggregatedStatus: aggregatedStatuses, Conditions: binding.Status.Conditions}
```

Comment:
If we introduce another field to `workv1alpha2.ResourceBindingStatus`, here will always update the status.
So, suggest:
```golang
	currentBindingStatus := binding.Status.DeepCopy()
	currentBindingStatus.AggregatedStatus = aggregatedStatuses
	meta.SetStatusCondition(&currentBindingStatus.Conditions, generateFullyAppliedCondition(binding.Spec.Clusters, aggregatedStatuses))
```